FHEM/14_SD_WS_Maverick.pm

```diff
@@ -196,51 +298,51 @@ sub SD_WS_Maverick_Attr(@)
 =begin html
 
 <a name="SD_WS_Maverick"></a>
-<h3>Wether Sensors protocol #7</h3>
+<h3>Weather Sensors protocol #47</h3>
 <ul>
   The SD_WS_Maverick module interprets temperature sensor messages received by a Device like CUL, CUN, SIGNALduino etc.<br>
   <br>
   <b>Known models:</b>
   <ul>
-    <li>Eurochon EAS800z</li>
-    <li>Technoline WS6750/TX70DTH</li>
+    <li>Maverick 732/733</li>
   </ul>
   <br>
-  New received device are add in fhem with autocreate.
+  New received device will be added in fhem with autocreate.
   <br><br>
 
   <a name="SD_WS_Maverick_Define"></a>
   <b>Define</b> 
   <ul>The received devices created automatically.<br>
-  The ID of the defice is the cannel or, if the longid attribute is specified, it is a combination of channel and some random generated bits at powering the sensor and the channel.<br>
-  If you want to use more sensors, than channels available, you can use the longid option to differentiate them.
+  Maverick generates a new ID on each time turned on. So it is not possible to link the hardware with the fhem-device. 
+  The consequence is, that only one Maverick could be defined in fhem.
   </ul>
   <br>
   <a name="SD_WS_Maverick Events"></a>
   <b>Generated readings:</b>
-  <br>Some devices may not support all readings, so they will not be presented<br>
   <ul>
-  	 <li>State (T: H:)</li>
-     <li>temperature (&deg;C)</li>
-     <li>humidity: (The humidity (1-100 if available)</li>
-     <li>battery: (low or ok)</li>
-     <li>channel: (The Channelnumber (number if)</li>
+  	 <li>State (Food: BBQ: )</li>
+     <li>temp-bbq (&deg;C)</li>
+     <li>temp-food (&deg;C)</li>
+     <li>messageType (6A at startup or rsync, otherwise 59)</li>
+     <li>checksum (experimental)</li>
   </ul>
   <br>
   <b>Attributes</b>
   <ul>
+    <li>inaktivityInterval <minutes (1-60)><br>
```

Comment:
okay, ich bin bis hier von Sekunden ausgegangen.

Da man in FHEM fast alles in Sekunden angibt, wäre das nicht besser es hier auch zu tun?

Reply:
habe es auf Sekunden geändert